nucypher/blockchain/eth/sol/source/contracts/UserEscrow.sol

```diff
@@ -92,109 +89,37 @@ contract UserEscrow is Ownable {
-    * @notice Withdraw available amount of tokens from the miners escrow to the user escrow
-    * @param _value Amount of token to withdraw
-    **/
-    function minerWithdraw(uint256 _value) public onlyOwner {
-        escrow.withdraw(_value);
-        emit WithdrawnAsMiner(owner, _value);
+        emit TokensWithdrawn(owner, _value);
     }
 
     /**
-    * @notice Lock some tokens or increase lock in the miners escrow
-    * @param _value Amount of tokens which should lock
-    * @param _periods Amount of periods during which tokens will be locked
+    * @notice Withdraw available ETH to the owner
     **/
-    function lock(uint256 _value, uint16 _periods) public onlyOwner {
-        escrow.lock(_value, _periods);
-        emit Locked(owner, _value, _periods);
-    }
-
-    /**
-    * @notice Divide stake into two parts
-    * @param _index Index of stake
-    * @param _newValue New stake value
-    * @param _periods Amount of periods for extending stake
-    **/
-    function divideStake(
-        uint256 _index,
-        uint256 _newValue,
-        uint16 _periods
-    )
-        public onlyOwner
-    {
-        escrow.divideStake(_index, _newValue, _periods);
-        emit Divided(owner, _index, _newValue, _periods);
-    }
-
-    /**
-    * @notice Confirm activity for future period in the miners escrow
-    **/
-    function confirmActivity() external onlyOwner {
-        escrow.confirmActivity();
-        emit ActivityConfirmed(owner);
-    }
-
-    /**
-    * @notice Mint tokens in the miners escrow
-    **/
-    function mint() external onlyOwner {
-        escrow.mint();
-        emit Mined(owner);
-    }
-
-    /**
-    * @notice Withdraw available reward from the policy manager to the user escrow
-    **/
-    function policyRewardWithdraw() public onlyOwner {
-        uint256 balance = address(this).balance;
-        policyManager.withdraw();
-        emit RewardWithdrawnAsMiner(owner, address(this).balance - balance);
-    }
-
-    /**
-    * @notice Withdraw available reward to the owner
-    **/
-    function rewardWithdraw() public onlyOwner {
+    function withdrawETH() public onlyOwner {
         uint256 balance = address(this).balance;
         require(balance != 0);
         owner.transfer(balance);
-        emit RewardWithdrawn(owner, balance);
+        emit ETHWithdrawn(owner, balance);
     }
 
     /**
-    * @notice Set the minimum reward that the miner will take in the policy manager
+    * @dev Fallback function send all requests to the target proxy contract
     **/
-    function setMinRewardRate(uint256 _minRewardRate) public onlyOwner {
-        policyManager.setMinRewardRate(_minRewardRate);
-        emit MinRewardRateSet(owner, _minRewardRate);
-    }
-
-    /**
-    * @notice Vote for the upgrade in the government contract
-    **/
-    function vote(bool _voteFor) public onlyOwner {
-        government.vote(_voteFor);
-        emit Voted(owner, _voteFor);
+    function () public payable onlyOwner {
```

Comment:
This is an anonymous function, or am I misunderstanding the syntax?

Reply:
it's fallback function, uses for receiving all unrecognised requests

Reply:
Ah - right, got it.